tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/hive/tpch/TpchTableDefinitions.java

```diff
@@ -22,7 +22,7 @@
 // TODO: move to separate module
 public class TpchTableDefinitions
 {
-    public static final double DEFAULT_SCALE_FACTOR = 0.01;
+    public static final double DEFAULT_SCALE_FACTOR = 1;
```

Comment:
Now it takes 12 minutes more. It finishes within 40 minutes. I have not noticed any OOM.